elements/pfe-jump-links/pfe-jump-links.ts

```diff
@@ -1,17 +1,143 @@
-import { PfeJumpLinksNav } from './pfe-jump-links-nav.js';
-export { PfeJumpLinksPanel } from './pfe-jump-links-panel.js';
-export { PfeJumpLinksNav };
+import { html, LitElement } from 'lit';
+import { customElement, property, queryAssignedElements } from 'lit/decorators.js';
 
-import { pfelement } from '@patternfly/pfe-core/decorators.js';
-import { customElement } from 'lit/decorators.js';
+import { ScrollSpyController } from '@patternfly/pfe-core/controllers/scroll-spy-controller.js';
+import { RovingTabindexController } from '../../core/pfe-core/controllers/roving-tabindex-controller.js';
```

Comment:
Should this be updated to pull from `@patternfly/pfe-core/controllers/roving-tabindex-controller.js` instead of `../../core/pfe-core/controllers/roving-tabindex-controller.js`?  & the same for line 6.

Reply:
Fixed in https://github.com/patternfly/patternfly-elements/pull/2283/commits/fe724763f7e34d451e43232c755387fcb5937c34